Podfile.lock

```diff
@@ -69,6 +70,7 @@ DEPENDENCIES:
   - SocketRocket (from `https://github.com/square/SocketRocket.git`, commit `954750c018`)
   - SSKeychain
   - TwistedOakCollapsingFutures (~> 1.0)
+  - UICKeyChainStore (from `Podspecs/UICKeyChainStore.podspec`)
```

Comment:
oh, didn't see that, that's fine. I was just using the same lib they did
originally for consistency. I have to go to bed and want to focus on
attachment stuff (audio and video) tomorrow. if you like you can add that
in if you know the API well. it is just in a few places. otherwise I will
do in the morning! thanks for the review on that good eye

On Mon, Jan 12, 2015 at 1:01 AM, Frederic Jacobs <notifications@github.com>
wrote:

> In Podfile.lock
> <https://github.com/WhisperSystems/Signal-iOS/pull/318#discussion-diff-22783992>
> :
>
> > @@ -69,6 +70,7 @@ DEPENDENCIES:
> >    - SocketRocket (from `https://github.com/square/SocketRocket.git` <https://github.com/square/SocketRocket.git>, commit `954750c018`)
> >    - SSKeychain
> >    - TwistedOakCollapsingFutures (~> 1.0)
> > +  - UICKeyChainStore (from `Podspecs/UICKeyChainStore.podspec`)
>
> No, I wouldn't do that. You can access the keychain items with SSKeychain
> APIs.
> The syntax of the API call is similar.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WhisperSystems/Signal-iOS/pull/318/files#r22783992>.
>



-- 
Dr. Corbett Moran
Rockets @ http://www.spacex.com
Physics @ http://www.ICS.uzh.ch  <http://www.ICS.uzh.ch>
Dev @ http://circleof6app.com
Dev @ https://whispersystems.org
www.christinecorbettmoran.com